### ISSUE TITLE
Add convert-to-pds subcommand

### ISSUE THREAD

**Issue by kmgill** [kmgill/mars-raw-utils#44]
Add a subcommand which will convert a calibrated image to PDS format. If this requires writing a whole VICAR support library, then lower the priority due to larger scope of work.

**Comment by kmgill** [comment]
Likewise, if a VICAR library _is_ written, then create a pds-to-png subcommand.

**Comment by sschmaus** [comment]
perhaps this functionality could be implemented via gdal

**Comment by kmgill** [comment]
Yeah, it can be done by gdal as it understands VICAR. Not as fun/interesting as writing my own implementation too. Vicar is pretty straightforward, though writing a serializer/deserializer (I'm researching using `serde` for this) for the PDS v<=3 PVL labels is a bit more complicated.
